var/spack/repos/builtin/packages/spfft/package.py

```diff
@@ -71,6 +71,15 @@ class Spfft(CMakePackage, CudaPackage, ROCmPackage):
     # before version 1.0.3
     patch("0001-fix-missing-limits-include.patch", when="@:1.0.2")
 
+    # Enable NVPL FFT
+    patch(
+        "https://github.com/eth-cscs/SpFFT/pull/64.patch?full_index=1",
+        when="@1.1.1",
+        sha256="570f56cb1f4a3e89b8f437c945e18749052c148aa228237e72b640d3f32dd027",
+    )
+    conflicts("@:1.1.0", when="^[virtuals=fftw-api] nvpl-fft")
```

Comment:
Is this conflict and patch still needed? I think it should work now with older versions of SpFFT as well.

Reply:
That's right. The patch is also no longer needed.